src/app.cpp

```diff
@@ -88,8 +88,40 @@ class simulationManager {
     }
 };
 
+class PythonKappaBarController : public baseController
```

Comment:
Can we move this to its own source and header file? I'd like to keep `app.cpp` pretty lean with just pybind definitions.